File: Palindrome.pl
print("Enter number or string to check for palindrome ");
$a=<STDIN>;
chomp($a);
$a= lc($a);
$r=0;
if ($a eq reverse($a))
{
    print("Is Pal")
}
else
{
    print("Isn't Pal")
}

print"Enter a number: ";
$n=<STDIN>;
$t=$n;
$s=0;
while($n>0)
{
$r=$n%10;
$s=($s*10)+$r;
$n=int($n/10);
}
if($t==$s)
{
print"Number is palindrome\n";
}
else
{
print"Number is not palindrome\n";
}

File: Palindrome.php
<html>
    <head>
        <link rel="stylesheet" href="style.css"/>
    </head>
    <body>
        <form method="POST">
            <table>
                <tr>
                    <th>Enter a string or number</th>
                    <td><input type="text" name="num" /><br/></td>
                </tr>
                <tr>
                    <td colspan="2"><input type="submit" class="Button" value="Check"/></td>
                </tr>
                <tr>
                    <td colspan="2">
        <?php
            if ($_SERVER["REQUEST_METHOD"]=="POST")
            {
                $n=$_POST['num'];
                $n=strtolower($n);
                $s=1;
                if ($n==strrev($n))
                {
                    echo "$n is Palindrome </td>";
                }
                else
                {
                    echo "$n is not Palindrome </td>";
                }
            }
        ?>
        </form>
    </body>
</html>